tests/upload_integration_test.go

```diff
@@ -92,46 +90,49 @@ func TestUploadFile(t *testing.T) {
 	}
 }
 
-// TestDownloadFile tests downloading a single file.
-func TestDownloadFile(t *testing.T) {
-	defer gock.Off() // Flush pending mocks after test execution
+// TestUploadFileSkykey tests uploading a file with either a skykey name or
+// skykey id set.
+func TestUploadFileSkykey(t *testing.T) {
```

Comment:
Do we have `TestDownloadFileSkykey` and `TestDownloadDirectorySkykey`?

Reply:
I'm not sure how to test this here. The portal will just send the original data if the skykey exists, right? Also, I haven't written tests for downloading directories yet.

Reply:
I thought we talked about how the SDK testing can't do the validation that the Skykey is valid, which is fine. Mostly these test would be to verify that the submitted values are handled properly right? Basically unit testing? Or am I still not understand the `gock` testing haha

Reply:
Yep that's right! I'll add `TestDownloadFileSkykey` sending skykey name and id